src/screens/LoginPage/LoginPage.spec.tsx

```
@@ -26,7 +26,9 @@
 import i18nForTest from 'utils/i18nForTest';
 import { BACKEND_URL } from 'Constant/constant';
 import useLocalStorage from 'utils/useLocalstorage';
-import { vi, beforeEach, expect, it, describe } from 'vitest';
+import { vi, beforeEach, expect, it, describe, Mock } from 'vitest';
+import { toast } from 'react-toastify';
+import { error } from 'console';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)